packit.spec

```diff
@@ -57,48 +35,44 @@ Requires:       rpm-build
 Requires:       rpmdevtools
 # Copying files between repositories
 Requires:       rsync
-%if 0%{?rhel}
-# rhbz#1968618 still not fixed for epel-8
-Requires:       python3-koji
-%endif
-%{?python_provide:%python_provide python3-%{real_name}}
 
-%description -n python3-%{real_name}
+%description -n python3-packit
 Python library for Packit,
 check out packit package for the executable.
 
 
 %prep
-%autosetup -n %{pypi_name}-%{version}
-# Remove bundled egg-info
-rm -rf %{pypi_name}.egg-info
+%autosetup -n packitos-%{version}
+
+
+%generate_buildrequires
+# The -w flag is required for EPEL 9's older hatchling
+%pyproject_buildrequires %{?with_tests:-x testing} %{?el9:-w}
 
-%if 0%{?rhel}
-# rhbz#1968618 still not fixed for epel-8
-sed -i -e 's|koji|# koji|' setup.cfg
-%endif
 
 %build
-%py3_build
+%pyproject_wheel
+
 
 %install
-%py3_install
-python3 setup.py --command-packages=click_man.commands man_pages --target %{buildroot}%{_mandir}/man1
+%pyproject_install
+%pyproject_save_files packit
+PYTHONPATH="%{buildroot}%{python3_sitelib}" click-man packit --target %{buildroot}%{_mandir}/man1
+
+install -d -m 755 %{buildroot}%{bash_completions_dir}
+cp files/bash-completion/packit %{buildroot}%{bash_completions_dir}/packit
 
-install -d -m 755 %{buildroot}%{_datadir}/bash-completion/completions
-cp files/bash-completion/packit %{buildroot}%{_datadir}/bash-completion/completions/packit
 
 %files
 %license LICENSE
 %{_bindir}/packit
 %{_mandir}/man1/packit*.1*
-%dir %{_datadir}/bash-completion/completions
-%{_datadir}/bash-completion/completions/%{real_name}
+%dir %{bash_completions_dir}
```

Comment:
```suggestion
```

This directory is owned by `filesystem`.

Reply:
I was not sure why the `%dir` was needed there good to know it was an artifact.